File: SqlFs/src/com/sss/sqlfs/SqlFsVersion.java
package com.sss.sqlfs;

public class SqlFsVersion 
{
   private SqlFsVersion() { }
   
   private static final String major = "0";
   private static final String minor = "10";
   private static final String build = "0";
    
   /**
    *  Get SqlFs version
    */
   public static String getSqlFsVersion()
   {
	  return major + "." + minor + "." + build;
   }
}

File: sqlfs#/SqlFsVersion.cs
namespace com.sss.sqlfs
{

	public class SqlFsVersion
	{
	   private SqlFsVersion()
	   {
	   }

	   private const string major = "0";
	   private const string minor = "10";
	   private const string build = "0";

	   /// <summary>
	   ///  Get SqlFs version
	   /// </summary>
	   public static string SqlFsVersion
	   {
		   get
		   {
			  return major + "." + minor + "." + build;
		   }
	   }
	}

}